templates/cli/lib/commands/command.js.twig

```diff
@@ -105,20 +105,35 @@ const {{ service.name | caseLower }}{{ method.name | caseUcfirst }} = async ({ {
         const streamFilePath = payload['{{ parameter.name }}'];
         let id = undefined;
 
+        let counter = 0;
         const totalCounters = Math.ceil(size / libClient.CHUNK_SIZE);
 
-        for (let counter = 0; counter < totalCounters; counter++) {
-                const start = (counter * libClient.CHUNK_SIZE);
-                const end = Math.min((((counter * libClient.CHUNK_SIZE) + libClient.CHUNK_SIZE) - 1), size);
-                const headers = {
+        const headers = {
 {% for parameter in method.parameters.header %}
-                        '{{ parameter.name }}': ${{ parameter.name | caseCamel | escapeKeyword }},
+            '{{ parameter.name }}': ${{ parameter.name | caseCamel | escapeKeyword }},
 {% endfor %}
 {% for key, header in method.headers %}
-                        '{{ key }}': '{{ header }}',
+            '{{ key }}': '{{ header }}',
+{% endfor %}
+        };
+
+{% for parameter in method.parameters.all %}
+{% if parameter.isUploadID %}
+        if({{ parameter.name | caseCamel | escapeKeyword }} != 'unique()') {
+                try {
+                        response = await client.call('get', path + '/' + {{ parameter.name }}, headers);
+                        counter = response.chunksUploaded;
+                } catch(e) {
+                }
```

Comment:
We use a convention of 4 spaces for each indent level. This looks like 8 spaces. Please check that. 

https://github.com/appwrite/sdk-generator/blob/9274cc934fbaa05a185551c8c643c40d736d561b/templates/cli/lib/commands/command.js.twig#L133-L162
Here as well, it's 8 spaces. Please look into it